pkg/controllers/termination/controller.go

```diff
@@ -84,7 +87,11 @@ func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	// 4. Drain node
 	drained, err := c.Terminator.drain(ctx, node)
 	if err != nil {
-		return reconcile.Result{}, fmt.Errorf("draining node %s, %w", node.Name, err)
+		if IsNodeDrainErr(err) {
+			c.Recorder.NodeFailedToDrain(node, err)
+		} else {
+			return reconcile.Result{}, fmt.Errorf("draining node %s, %w", node.Name, err)
+		}
```

Comment:
Also -- we shouldn't need to include node name, since we have `ctx = logging.WithLogger(ctx, logging.FromContext(ctx).Named(controllerName).With("node", req.Name))`